content/blog/provisioning-mks-clusters-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,238 @@
+
+![](/img/cluster-worker.png)
+
+6. **Review cluster details**
+
+Skip the step for automation settings, then the cluster review page appears:
+
+![](/img/cluster-review.png)
+
+Click _**Complete**_ to begin provisioning the MKS cluster:
+
+![](/img/cluster-provisioning.png)
+
+## Verify MKS cluster
+
+After a few minutes, the cluster *'mks-demo'* is created using the specified cluster layout: *MKS Kubernetes 1.31 Cluster on Ubuntu 22.04*. 
+
+![](/img/cluster-status.png)
+
+## Access MKS cluster
+
+Click the *'mks-demo'* cluster to view its details under the *Summary* tab:
+
+![](/img/cluster-details.png)
+
+Navigate to the _**Control**_ tab and run the command *'kubectl get nodes'* to view the cluster's node information:
+
+![](/img/cluster-nodes.png)
+
+In line with the cluster type, *MKS Kubernetes 1.31 Cluster on Ubuntu 22.04.*, the *'mks-demo'* cluster consists of one master and three workers.
+
+## Run daily cluster operations
+
+From the provisioned MKS cluster, the _**Actions**_ menu provides a curated set of supported operations that simplify and streamline day-to-day cluster management. From downloading kubeconfig to scaling the cluster and performing upgrade, these built-in actions help automate key cluster operations, making cluster administration faster, easier and more consistent. 
+
+![](/img/cluster-actions.png)
+
+The following sections explore how to perform these day-to-day cluster operations.
+
+* **View and download kube config**
+
+From the *'mkd-demo'* cluster, click _**Actions**_:
+
+![](/img/view-kubeconfig-menu.png)
+
+Click *View Kube Config* to show the Kube config of the cluster: 
+
+![](/img/view-kubeconfig.png)
+
+Save the Kube config content to a file, for example, *'mks-demo.kubeconfig'* on a *Linux* client as shown below. This file can then be used with the *kubectl* CLI or the *Helm* tool to access the MKS cluster and deploy applications using either YAML manifests or Helm charts. 
+
+![](/img/kubectl-console.png)
+
+* **Add addtional worker**
+
+In K8s, adding new workers to an existing cluster involves setting up self-registration, provisioning VM instances, and integrating them into the cluster. This process can be time-consuming and often demands custom automation scripts to streamline the workflow.
+
+In MKS, this operation is supported by simply clicking _**Actions**_ from the cluster:
+
+![](/img/add-worker.png)
+
+Click *Add VMware Kubernetes Worker* to begin adding new worker to the cluster. Then follow the steps outlined below:
+
+1. Specify *NAME* and the optional *DESCRIPTION*:
+
+![](/img/cluster-add-worker-name.png)
+
+2. Select *PLAN* and configure *VOLUMES* and *NETWORKS*: 
+
+![](/img/cluster-add-worker-config.png)
+
+If you want to add multiple workers at once, set the desired value in the *NUMBER OF WORKERS* field. 
+
+3. Review worker details
+
+Skip the step for automation settings, then the worker review page appears:
```

Comment:
Suggestion:

Skip the step for **Automation** settings, then the worker review page appears: